Makefile

```diff
@@ -291,7 +291,6 @@ TOOLS = govulncheck \
 		jq \
 		kubectl \
 		kustomize \
-		yq \
```

Comment:
Instead of removing it, how about we maintian a different target called `ci-tools` and have 

```
tools: ci-tools 
```
YQ is quite useful since we have a lot of yaml files in `manifests`

Reply:
I'm all for a new target `ci-tools` is ok - but it doesn't encompass the development side.

could we not just have a `deps` target that installs all the depdendencies and centralise all the dependencies/tools in one repo...

From a dev experience POV - it would make the onboarding experience a lot smoother IMHO. Right now we rely on the developer to manually run a command to install the tools they need after finding the appropriate reference in the Readme rather than automating this step

Reply:
My honest feedback is the developer experience for Kepler right now could be greatly improved. It's not just about CI...